python/src/TestResult_doc.i.in

```diff
@@ -3,9 +3,9 @@
 
 .. warning::
 
-    Tests results are not intended to be created manually. They are returned
-    by the statistical tests implemented in the :mod:`~openturns.stattests`
-    subpackage. Constructor is therefore intentionally not documented.
+    Tests results are not intended to be created manually.
```

Comment:
As a side comment unrelated to this PR, the fact that `TestResult` is intentionally undocumented is a strange choice. There is no particular reason to prevent users from creating _any_ object.

Reply:
yes I agree